larq_compute_engine/mlir/python/graphdef_tfl_flatbuffer.cc

```diff
@@ -95,25 +106,25 @@ pybind11::bytes ConvertGraphDefToTFLiteFlatBuffer(
           default_ranges.cast<std::pair<double, double>>();
     }
   }
-  mlir::PassManager pm(&context);
+
+  mlir::PassManager pm(&context, mlir::OpPassManager::Nesting::Implicit);
+  tensorflow::SetCrashReproducer(pm);
+
   tensorflow::AddTFToLCETFLConversionPasses(
       quant_specs, &pm, target, experimental_enable_bitpacked_activations);
 
   // Convert back to outlined while format for export back to flatbuffer.
   pm.addPass(mlir::TFL::CreateWhileOutlinePass());
   pm.addPass(mlir::TFL::CreateRuntimeVerifyPass());
 
-  if (failed(pm.run(*module.ValueOrDie()))) {
-    throw std::runtime_error("Could not complete conversion passes.");
-  }
-
-  TruncateOpOrArgLocNameMapper op_or_arg_name_mapper;
```

Comment:
@AdamHillier Do we not need this truncation anymore, or is it now handled within the conversion function?

We added it in https://github.com/larq/compute-engine/pull/349/commits/df24c2b7f9b1f01631acab8c89f72160106afb42, not sure if we still want to keep this.

Sorry for not spotting this earlier.

Reply:
Ah sorry, I missed that too :p Thanks for the PR :)